api/trivia.js

```diff
@@ -8,195 +8,262 @@ const shuffleArray = arr => arr
 
 module.exports = function(app) {
 
+  
   app.get("/api/trivia/getQuestions", async(req, res) => {
+
+    // Question 1.
+   // Template: PERSON was born in ...
     const topBiosInOccupation = await axios.get("https://api.pantheon.world/person_ranks?occupation_rank=lt.7&select=id,name,occupation,birthyear,bplace_geonameid,bplace_country,deathyear,dplace_country").then(result => result.data).catch(e => (console.log("[trivia] Pantheon DB Error:", e), {data: []}));
-    // return res.json(topBiosInOccupation);
+
     const ezOccupations = ["ACTOR", "SOCCER PLAYER", "FILM DIRECTOR", "EXPLORER", "COMPOSER", "ASTRONOMER", "COMIC ARTIST", "ARTIST", "MUSICIAN", "SINGER", "ASTRONAUT", "BUSINESSPERSON", "PHILOSOPHER", "INVENTOR"];
     const biosInEZOccupations = topBiosInOccupation.filter(bio => ezOccupations.includes(bio.occupation));
+
     const randomEZBioIndex = Math.floor(Math.random() * biosInEZOccupations.length);
     const randomEZBio = biosInEZOccupations.splice(randomEZBioIndex, 1)[0]; // Pop the item from the array
     const wrongCountries = [...new Set(biosInEZOccupations.map(d => d.bplace_country))].filter(d => Boolean(d) && d !== randomEZBio.bplace_country);
-    const wrongCountry1 = wrongCountries[Math.floor(Math.random() * wrongCountries.length)];
-    const wrongCountry2 = wrongCountries[Math.floor(Math.random() * wrongCountries.length)];
-    const wrongCountry3 = wrongCountries[Math.floor(Math.random() * wrongCountries.length)];
+    const wrongCountry1 = wrongCountries[Math.floor(Math.random() * (wrongCountries.length-1)) + 1];
+    const wrongCountry2 = wrongCountries[Math.floor(Math.random() * (wrongCountries.length-1)) + 1];
+    const wrongCountry3 = wrongCountries[Math.floor(Math.random() * (wrongCountries.length-1)) + 1];
     const q1 = {
-      question: `Where was ${randomEZBio.occupation} ${randomEZBio.name} born?`,
+      question: `${randomEZBio.name} was born  in ...`,
       answer_a: wrongCountry1,
       answer_b: wrongCountry2,
       answer_c: randomEZBio.bplace_country,
       answer_d: wrongCountry3,
       correct_answer: "c"
     };
+    console.log("q1");
```

Comment:
Do we need these console statements? This will fill up the logs quite quickly if we leave them.